core/extension/extension_api_dump.cpp

```diff
@@ -846,6 +846,10 @@ Dictionary NativeExtensionAPIDump::generate_extension_api() {
 					if (F.name.begins_with("_")) {
 						continue; //hidden property
 					}
+					if (F.name.find("/") >= 0) {
+						// Ignore properties with '/' (slash) in the name. These are only meant for use in the inspector.
+						continue;
+					}
```

Comment:
I'm not sure if that's always true.

At least some of them seem to be real but badly named properties:
```
doc/classes/AudioEffectDelay.xml:               <member name="feedback/active" type="bool" setter="set_feedback_active" getter="is_feedback_active" default="false">
doc/classes/AudioEffectDelay.xml:               <member name="feedback/delay_ms" type="float" setter="set_feedback_delay_ms" getter="get_feedback_delay_ms" default="340.0">
doc/classes/AudioEffectDelay.xml:               <member name="feedback/level_db" type="float" setter="set_feedback_level_db" getter="get_feedback_level_db" default="-6.0">
doc/classes/AudioEffectDelay.xml:               <member name="feedback/lowpass" type="float" setter="set_feedback_lowpass" getter="get_feedback_lowpass" default="16000.0">
doc/classes/AudioEffectDelay.xml:               <member name="tap1/active" type="bool" setter="set_tap1_active" getter="is_tap1_active" default="true">
doc/classes/AudioEffectDelay.xml:               <member name="tap1/delay_ms" type="float" setter="set_tap1_delay_ms" getter="get_tap1_delay_ms" default="250.0">
doc/classes/AudioEffectDelay.xml:               <member name="tap1/level_db" type="float" setter="set_tap1_level_db" getter="get_tap1_level_db" default="-6.0">
doc/classes/AudioEffectDelay.xml:               <member name="tap1/pan" type="float" setter="set_tap1_pan" getter="get_tap1_pan" default="0.2">
doc/classes/AudioEffectDelay.xml:               <member name="tap2/active" type="bool" setter="set_tap2_active" getter="is_tap2_active" default="true">
doc/classes/AudioEffectDelay.xml:               <member name="tap2/delay_ms" type="float" setter="set_tap2_delay_ms" getter="get_tap2_delay_ms" default="500.0">
doc/classes/AudioEffectDelay.xml:               <member name="tap2/level_db" type="float" setter="set_tap2_level_db" getter="get_tap2_level_db" default="-12.0">
doc/classes/AudioEffectDelay.xml:               <member name="tap2/pan" type="float" setter="set_tap2_pan" getter="get_tap2_pan" default="-0.4">
doc/classes/CharacterBody2D.xml:                <member name="collision/safe_margin" type="float" setter="set_safe_margin" getter="get_safe_margin" default="0.08">
doc/classes/CharacterBody3D.xml:                <member name="collision/safe_margin" type="float" setter="set_safe_margin" getter="get_safe_margin" default="0.001">
doc/classes/Joint3D.xml:                <member name="collision/exclude_nodes" type="bool" setter="set_exclude_nodes_from_collision" getter="get_exclude_nodes_from_collision" default="true">
doc/classes/Joint3D.xml:                <member name="nodes/node_a" type="NodePath" setter="set_node_a" getter="get_node_a" default="NodePath(&quot;&quot;)">
doc/classes/Joint3D.xml:                <member name="nodes/node_b" type="NodePath" setter="set_node_b" getter="get_node_b" default="NodePath(&quot;&quot;)">
doc/classes/Joint3D.xml:                <member name="solver/priority" type="int" setter="set_solver_priority" getter="get_solver_priority" default="1">
```

Reply:
I'm fixing those in #44876.